app/pages/resetPasswordSuccess/page.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+import Messages from "./messages";
+
+export default function resetPasswordSucess() {
+  return (
+    <div className="flex justify-center items-center">
+      <div className="flex-1 flex flex-col w-full max-w-md px-8 py-8 gap-2">
+        <>
+          <form
+            className="flex-1 flex flex-col w-full justify-center gap-2 text-foreground"
+            action="/auth/password-reset"
+            method="post"
+          >
+            <label className="text-md" htmlFor="password">
+              New Password
+            </label>
+            <input
+              id="password"
+              className="rounded-md px-4 py-2 bg-inherit border mb-6"
+              type="password"
+              name="password"
+              placeholder="••••••••"
```

Comment:
Hrm, this placeholder seems a little odd. I’ll need to run it to form an opinion (I’m mid lecture at the moment), but I wonder if this looks like unexpected behaviour.

Since it’s already labelled, might get away with no placeholder at all.

Reply:
That was there from the last group for the login/signup pages so kept the consistency.